SofaKernel/SofaFramework/SofaMacros.cmake

```diff
@@ -567,7 +570,7 @@ macro(sofa_install_targets package_name the_targets include_install_dir)
         get_target_property(target_sources ${target} SOURCES)
         #list(FILTER ${target_sources} INCLUDE REGEX ".*\.h\.in$") # CMake >= 3.6
         foreach(filepath ${target_sources})
-            if(${filepath} MATCHES ".*\.h\.in$")
+            if(filepath AND ${filepath} MATCHES ".*.h.in$")
```

Comment:
No no...this is not the correct fix. See #1161  for the proper one.

Reply:
Ok thanks, I'll revert this :+1: